src/lib/components/Ideogram.react.js

```diff
@@ -517,128 +513,127 @@ Ideogram.propTypes = {
-     * for homology.
+     * Whether to include abbreviation species name in chromosome label. Used for homology.
      */
     fullChromosomeLabels: PropTypes.bool,
 
     /**
-     * Whether annotations should be filterable.
+     * The resolution of cytogenetic bands to show for each chromosome.
+     * The quantity refers to approximate value in bands per haploid set (bphs).
+     * One of 450, 550, or 850.
      */
-    filterable: PropTypes.number,
+    resolution: PropTypes.number,
 
     /**
-     * Provide local JSON organism into this prop from a local user JSON file.
-     * DataDir must not be initialized.
+     * Whether annotations should be filterable.
      */
-    localOrganism: PropTypes.object,
+    filterable: PropTypes.bool,
 
     /**
-     * Dash event callback for mousing over data.
+     * The orientation of chromosomes on the page.
      */
-    onMouseOver: PropTypes.func,
+    orientation: PropTypes.oneOf(['vertical', 'horizontal']),
 
     /**
-     * Organism(s) to show chromosomes for. Supply organism's name as a string (e.g. "human") or
-     * organism's NCBI Taxonomy ID (taxid, e.g. 9606) to display chromosomes from a single organism,
-     * or an array of organisms' names or taxids to display chromosomes from multiple species.
+     * The ploidy - number of chromosomes to depict for each chromosome set.
      */
-    organism: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    ploidy: PropTypes.number,
+
     /**
-     * The orientation of chromosomes on the page.
+     * Description of ploidy in each chromosome set in terms of ancestry composition.
      */
-    orientation: PropTypes.string,
+    ploidyDesc: PropTypes.arrayOf(PropTypes.object),
 
     /**
-     * Callback function to invoke after chromosome has rotated. (React)
+     * A map associating ancestor labels to colors. Used to color
+     * chromosomes from different ancestors in polyploid genomes.
      */
-    onDidRotate: PropTypes.func,
+    ancestors: PropTypes.object,
 
     /**
-     * Callback function to invoke when annotations are drawn. (React)
+     * List of objects describing segments of recombination among chromosomes in a chromosome set.
      */
-    onDrawAnnots: PropTypes.func,
+    rangeSet: PropTypes.arrayOf(PropTypes.object),
 
     /**
-     * Callback function to invoke when chromosomes are loaded,
-     * i.e. rendered on the page. (React)
+     * Whether chromosomes are rotatable on click.
      */
-    onLoad: PropTypes.func,
+    rotatable: PropTypes.bool,
 
     /**
-     * The ploidy - number of chromosomes to depict for each chromosome
-     * set.
+     * Dash callback that returns True if rotated, and false if not.
+     *
      */
-    ploidy: PropTypes.number,
+    rotated: PropTypes.bool,
 
     /**
-     * Description of ploidy in each chromosome set in terms of
-     * ancestry composition.
+     * Useful for omitting chromosome Y in female mammals.
```

Comment:
Apparently, the XY sex-determination system exists also in species which are not mammals! /cc @NadiaTahiri

Reply:
The flower represents the reproductive organ in flowering plants (_i.e._ _angiosperms_). 
The stamens produce the pollen grains that contain the **male** reproductive cells, 
while the pistil is where the ovules are produced, so **female** reproductive
_e.g._ Fertilization is usually cross-fertilized, that is, in most cases, a plant can not self-pollinate.
Source: [wikipedia](https://en.wikipedia.org/wiki/Plant_reproductive_morphology)

In conclusion, we can't talk X/Y sex determination system like animal side because is correlated to the organ 
@TahiriNadia here in my GitHub account which I use the other one is my old version ;-)

Reply:
And **Bacteria** is most complex because they may exchange genetic information

Reply:
Thanks, @TahiriNadia! I wasn't suggesting we should include plants; there are *animals* which are not *mammals*. Would it make more sense to write "female animals" rather than "female mammals"?

Reply:
Yes, you definitively can use `female animals` rather than `female mammals` is more strict on this second case.

Reply:
And the formal notation is `specie` instead of `organism`

Reply:
Thanks for sharing your expertise, @TahiriNadia! We'll include these improvements in a subsequent PR.

Reply:
You're welcome @mkcor